### ISSUE TITLE
23rd hour of day time series does not appear

### ISSUE THREAD

**Issue by arthurmacapagal** [anitagraser/TimeManager#162]
I am trying to show create the accidents in Paris as they occur by hour through time manager (showing through heatmaps). Everything works great until the last hour of the day, when the data for the 23h does not show, but rather just a blank screen. 

The data comes from [Paris Open Data](http://opendata.paris.fr/explore/dataset/accidentologie/export/?dataChart=eyJxdWVyaWVzIjpbeyJjb25maWciOnsiZGF0YXNldCI6ImFjY2lkZW50b2xvZ2llIiwib3B0aW9ucyI6e319LCJjaGFydHMiOlt7InR5cGUiOiJsaW5lIiwiZnVuYyI6IkFWRyIsInlBeGlzIjoiZGVwdCIsImNvbG9yIjoiIzY2YzJhNSJ9XSwieEF4aXMiOiJkYXRlIiwibWF4cG9pbnRzIjoiIiwidGltZXNjYWxlIjoieWVhciIsInNvcnQiOiIifV0sInRpbWVzY2FsZSI6InllYXIifQ%3D%3D&location=7,49.67201,2.07533) 

and I created a new column, grouping all the times by hour , and this is the column I use in time manager.

I am using QGIS 2.8.3 on a Mac OS 10.10.5

Thank you for any help!

![image](https://cloud.githubusercontent.com/assets/18381001/14411342/8e6ff01c-ff45-11e5-8f3d-c0bd1711cb5e.png)

![image](https://cloud.githubusercontent.com/assets/18381001/14411345/a93e1658-ff45-11e5-91fd-3cf8d146b894.png)

**Comment by carolinux** [comment]
Can you share the modified time data that you are loading?
Bests,
Karolina

**Comment by carolinux** [comment]
Although I think I know what's happening - instead of having the time frame size set to 1 hour, please set it to 59 minutes (if it's OK to lose one minute). The way you are using it, Time Manager is essentially trying to do the following query - I want all the data for hours >=23:00 and hours<00:00, which (numerically) returns nothing.

But if you make it so that its hours>=23:00 and hours<23:59, it will work.

**Comment by arthurmacapagal** [comment]
Thanks for the quick reply and suggestion!
I tried your suggestion, but the last frame still showed up as empty, and also, the result is a loss of 23 minutes by the final slide/frame.

Please find attached a csv of the data I am using. I created column Time_2 which I am using for in time manager
[Data Excelv.2.xlsx](https://github.com/anitagraser/TimeManager/files/211849/Data.Excelv.2.xlsx)

**Comment by carolinux** [comment]
Ok, so Time Manager doesn't fully support this particular case when the time format is just hour, minute, second. When I use your data and I go to 23:00:00 and set the interval to 59 minutes, I do get data, so I am not sure what you mean by "the final frame is empty" - perhaps you mean the one after that. Anyway, a workaround, I would use would be to add a date the Time_2 column  - make it be like: 2012-01-01 23:00:00 (for all the hours). That way the query will be OK.

Incidentally, from the dataset, I am not sure your data is formatted correctly - I can't tell from the excel whether you are using the hour format as 09:00:00 or 9:00:00. The former is the correct one.

**Comment by arthurmacapagal** [comment]
Thank you again for all of the help! The formatting is the former (i.e. in the form of 09:00:00). I tried the work around that you suggested, by adding the date, but for some reason it did not work. I did however find another 'work around'. In this case, since Time Manager doesn't fully support this particular time format, I changed the hours to be minutes ex. 23:00:00 --> 00:23:00 and then changed the interval to be 1 minute and the format of the time stamp to show just minutes and seconds (ex. 23:00) . This does the trick! 

Thank you again for all the help!!
